assets/js/components/EntityHeader.js

```diff
@@ -38,18 +38,21 @@ import {
  */
 import Data from 'googlesitekit-data';
 import ViewContextContext from './Root/ViewContextContext';
-import { VIEW_CONTEXT_PAGE_DASHBOARD } from '../googlesitekit/constants';
 import Button from './Button';
 import { CORE_SITE } from '../googlesitekit/datastore/site/constants';
 import BackspaceIcon from '../../svg/icons/keyboard-backspace.svg';
 import { CORE_LOCATION } from '../googlesitekit/datastore/location/constants';
 import Link from './Link';
 import { shortenURL } from '../util/urls';
 import { trackEvent } from '../util';
+import useDashboardType, {
+	DASHBOARD_TYPE_ENTITY,
+} from '../hooks/useDashboardType';
 const { useSelect, useDispatch } = Data;
 
 const EntityHeader = () => {
 	const viewContext = useContext( ViewContextContext );
+	const dashboardType = useDashboardType( ViewContextContext );
```

Comment:
`useDashboardType` does not take any argument so this is not needed.

Reply:
Yikes - thanks for this. This is what happens when you work on an issue, take a break from it and come back to it one week later. 😄